.github/workflows/workflow-lint-test-python.yml

```diff
@@ -41,3 +61,39 @@ jobs:
         PGPASSWORD: ${{ secrets.PGPASSWORD }}
         PGPORT: ${{ secrets.PGPORT }}
         PGUSER: ${{ secrets.PGUSER }}
+
+    - name: Compare coverage with threshold
+      id: compare_values
+      run: |
+        if [ $(echo "$CURRENTCOVERAGE > ${{ inputs.threshold }}" | bc) -eq 1 ]; then
+          echo "update_value=true" >> $GITHUB_OUTPUT
+          echo "::set-output name=current_coverage::$CURRENTCOVERAGE"
+        else
+          echo "update_value=false" >> $GITHUB_OUTPUT
+        fi
+      shell: /usr/bin/bash -e {0}
+      env:
+        CURRENTCOVERAGE: ${{ env.CURRENTCOVERAGE }}
+
+    - name: Update file
+      if: steps.compare_values.outputs.update_value == 'true'
+      uses: richardrigutins/replace-in-files@v2
+      with:
+        files: '.github/workflows/gcp-deployment.yml'
```

Comment:
this wouldn't work as multiple different projects can have their own coverage. this is changing the threshold for each. 

1) check fails when new coverage is smaller than the parameter coverage
2) check should also fails if delta from old to new coverage is more than +X%, requiring developer to modify manually the threshold in the CALLING github workflow